editor/src/clj/editor/settings_core.clj

```diff
@@ -188,9 +209,24 @@
 (defn- category->str [category settings]
   (s/join "\n" (cons (str "[" category "]") (map setting->str settings))))
 
+(defn split-multi-line-setting [category key settings]
+  (let [path (seq [category key])]
+    (if-let [comma-separated-setting (get-setting settings path)]
+      (let [comma-separated-setting-parts (seq (non-blank (s/split comma-separated-setting #",")))
+            comma-separated-setting-count (count comma-separated-setting-parts)
+            cleared-settings (clear-setting settings path)]
+        (reduce
+          (fn [settings i]
+            (set-setting settings (seq [category (str key "." i)]) (nth comma-separated-setting-parts i)))
+          cleared-settings
+          (range 0 comma-separated-setting-count)))
+      settings)))
+
```

Comment:
We can remove the `(seq ...)` calls here as well. Three places.

Reply:
But we don't split every setting with a comma? We explicitly split project.dependencies only.